docs/core/install/windows.md

```diff
@@ -388,6 +388,40 @@ Microsoft provides images that are tailored for specific scenarios. For example,
 
 For more information about using .NET in a Docker container, see [Introduction to .NET and Docker](../docker/introduction.md) and [Samples](https://github.com/dotnet/dotnet-docker/blob/main/samples/README.md).
 
+## Troubleshooting
+
+After installing .NET, you may run into problems trying to run .NET. This section collects those common problems and provides solutions.
+
+- [Can't find/run/locate .NET after installing](#cant-findrunlocate-net-after-installing)
+
+### Can't find/run/locate .NET after installing
+
```

Comment:
should you show what error message they get? Here's an example. (it would be good to confirm the latest)
![image](https://user-images.githubusercontent.com/12971179/153519729-3ecd00ce-1508-46e5-9e1f-158c24a9293c.png)

Reply:
I wasn't sure if this was the exact error message. Looking through some of the linked issues I found a few different "affects" people had, some of them this error, others just having conflicts with no error, just knowing they're going to the wrong SDK or something.

Reply:
I'd still add this one as an example. People will search for the error message and then would find this docs page if the text is part of the article not as an image.